src/main/Monorepo/Build.php

```diff
@@ -90,6 +90,7 @@ public function build($rootDirectory, $optimize = false, $noDevMode = false, $cl
                 $localRepo,
                 $mainPackage,
                 $installationManager,
+                $composer->getLocker(),
                 'composer',
                 $optimize
```

Comment:
See https://github.com/composer/composer/commit/cb363b0e84459b8d280d0ff6b5a7d4ccde93420d

You can do this to be compatible with all Composer versions except 2.6.4 which will break but I think that's ok as it was breaking anyway.

```suggestion
                'composer',
                $optimize,
                null,
                $composer->getLocker()
```